test/logging/logging.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2018 The Knative Authors
-
```

Comment:
nit: the boiletplate **does** contain the blank lines you removed

Reply:
updated.